bin/riemann-elasticsearch

```diff
@@ -36,7 +37,7 @@ class Riemann::Tools::Elasticsearch
   end
 
   def health_url
-    "http://#{options[:es_host]}:#{options[:es_port]}/_cluster/health"
+    "http://#{options[:es_host]}:#{options[:es_port]}/#{options[:path_prefix]}/_cluster/health"
```

Comment:
Normally they work, but better either test or parse it to single? Not sure
how to communicate the prefix format in a good way to the user...
On May 29, 2015 8:30 PM, "Kyle Kingsbury" <notifications@github.com> wrote:

> In bin/riemann-elasticsearch
> <https://github.com/aphyr/riemann-tools/pull/115#discussion_r31351629>:
>
> > @@ -36,7 +37,7 @@ class Riemann::Tools::Elasticsearch
> >    end
> >
> >    def health_url
> > -    "http://#{options[:es_host]}:#{options[:es_port]}/_cluster/health"
> > +    "http://#{options[:es_host]}:#{options[:es_port]}/#{options[:path_prefix]}/_cluster/health"
>
> Is a double slash (//) gonna work? Looks like the default is an empty
> string, so I want to make sure we don't break existing users.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aphyr/riemann-tools/pull/115/files#r31351629>.
>